test/index-removeItems.test.js

```diff
@@ -106,9 +106,9 @@ describe("Testing backoff", function () {
 					UnprocessedItems: { test: items.splice(0, (Math.random() * 2) | 0) }
 				});
 
-			database.removeItems("test", items).then(function () {
-				done();
-			});
+			const result = await database.removeItems("test", items);
+
+			expect(result).toHaveLength(8);
```

Comment:
❓ Just curious on where we get the `8` items?

Reply:
I just looked at the data returned but I also have no idea. This whole "unit test" doesn't need an expectations so just wanted to add something so I didn't need to add an eslint exception to the file.

As it isn't important what it is checking I have updated this to `expect(result).toBeDefined()`

Reply:
Nice resolution, the `expect(result).toBeDefined()` is much better than the length of `8`.